localgovdrupal/localgov_base
# Uses logical properties for fields.css file

Closes #627 

## What does this change?

Rewrites the fields.css file to use logical properties. By doing this we can cut down a lot on how much CSS we write and send to the browser.

## How to test

Only inline field labels are affected by this file, so set a field label to inline and make sure it works the same as currently.

---
Thanks to [Big Blue Door](https://www.bigbluedoor.net/) for sponsoring my time to work on this.
